README.md

```diff
@@ -16,3 +32,155 @@ pip install cookiecutter
+under the `tool.typos.default.extend-words` key.
+
+```toml
+[tool.typos.default.extend-words]
+mke = 'mke'
+```
+
+For further configuration, refer to the [typos docs][typos-docs].
+
+### Static type checker - Mypy
+
+Change the configuration in `pyproject.toml` file, for further information refer to
+the [documentation][mypy-config].
+
+### Documentation - Sphinx
+
+Configuration for Sphinx is in the `docs/source/conf.py` file, for further info refer
+to the [Sphinx documentation][sphinx-docs]. The default configuration uses
+[furo theme][furo] and [ReadTheDocs][rtd] to host the documentation, you can
+change the configuration using `/docs/source/conf.py` and `.readthedocs.yml`
+file.
+
+### Testing - pytest and pytest-cov
+
+Generate a coverage report using `pytest-cov` and uploads it to codecov.io.
+
+## CI/CD - GitHub Actions
+
+Here are the GitHub Actions Secrets that need to be included in the repository
+settings:
+
+- `PYPI_PASSWORD`: The password for the PyPI account.
+- `CODECOV_TOKEN`: The token for the codecov.io account.
+- `AUTO_UPDATE_GITHUB_TOKEN`: The token for the create a PR to update the template.
+
+## Disclamer
+
+Some of the links and images have been hardcoded with `ELIXIR Cloud & AAI`'s
+assets in the documentation, please update them manually if needed. These
+assets **MUST** be included for projects owned by ELIXIR Cloud & AAI, but
+**MUST NOT** be included for projects that are not (personal projects, projects
+owned by other orgs).
+
+**Only`markdown` files and `images/` directory need to be changed.**
+
+## Contributing
+
+This project is a community effort and lives off _your_ contributions, be it in
+the form of bug reports, feature requests, discussions, fixes or any other form
+of contribution!
+
+Please refer to the guidelines available at [`CONTRIBUTING.md`][contributing] if
+you are interested in contributing.
+
+## Code of Conduct
+
+Please mind the code of conduct described in
+[`CODE_OF_CONDUCT.md`][code-of-conduct] for all interactions with the community.
+Please be nice to one another! :)
+
+If you experience any unacceptable behavior by any member of the community,
+please refer to the contact method specified in
+[`CODE_OF_CONDUCT.md`][code-of-conduct] to report the incident to the community
+leaders.
+
+## Versioning
+
+The project adopts the [semantic versioning][semver] scheme for versioning.
+Currently the service is in a pre-release stage, so changes to the API,
+including breaking changes, may occur at any time without further notice.
+
+## License
+
+This project is distributed under the [Apache License 2.0][badge-license-url], a
+copy of which is also available in [`LICENSE`][license].
+
+## Contact
+
+The project is maintained by [ELIXIR Cloud & AAI][elixir-cloud-aai], a Driver
+Project of the [Global Alliance for Genomics and Health (GA4GH)][ga4gh], under
+the umbrella of the [ELIXIR] [Compute Platform][elixir-compute].
```

Comment:
I would add the `[ELIXIR][elixir]` ref explicitly - otherwise there's a change it'll diverge, which is difficult to catch.

Reply:
md-formatter that has been set in this repo auto remove this.

Reply:
Okay, never mind then